samples/meetings-token-app/nodejs/client/src/components/TabConfig.js

```diff
@@ -13,44 +13,45 @@ import * as microsoftTeams from "@microsoft/teams-js";
  */
 class TabConfig extends React.Component {
 
-    render() {
-      microsoftTeams.app.initialize().then(() => {
-        /**
-         * When the user clicks "Save", save the url for your configured tab.
-         * This allows for the addition of query string parameters based on
-         * the settings selected by the user.
-         */
-        microsoftTeams.pages.config.registerOnSaveHandler((saveEvent) => {
-            microsoftTeams.pages.config.setConfig({
-            "suggestedDisplayName": "Token App",
-            "entityId": "Test",
-            "contentUrl": window.location.origin + "/",
-            "websiteUrl": window.location.origin + "/"
-          });
-          saveEvent.notifySuccess();
+  componentDidMount() {
+    microsoftTeams.app.initialize().then(() => {
+      /**
+       * When the user clicks "Save", save the url for your configured tab.
+       * This allows for the addition of query string parameters based on
+       * the settings selected by the user.
+       */
+      microsoftTeams.pages.config.registerOnSaveHandler((saveEvent) => {
+        microsoftTeams.pages.config.setConfig({
+          "suggestedDisplayName": "Token App",
+          "entityId": "Test",
```

Comment:
Can't we put some meaningful entityId rather than "Test" ?
Also, change at the other dependent places.

Reply:
Done